cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -963,8 +956,8 @@ def test_monitoring_endpoint_validation(self):
         self.rjObj = RadosJSON(['--rbd-data-pool-name=abc', '--format=json'])
         self.rjObj.cluster = DummyRados.Rados()
 
-        valid_ip_ports = [("10.22.31.131", "9283"),
-                          ("10.177.3.81", ""), ("", ""), ("", "9283")]
+        valid_ip_ports = [("10.22.31.131", "3534"),
+                          ("10.177.3.81", ""), ("", ""), ("", "9092")]
```

Comment:
How are these ports used? If we have the ability to override the default in Rook, will we just use the default unless it's overridden?

Reply:
They are not used, they are just unit tests.